modules/ROOT/pages/backup-restore/online-backup.adoc

```diff
@@ -308,19 +308,31 @@ When using the `neo4j-admin database backup` command, you can configure the back
 For more information on how to configure SSL in Neo4j, see xref:security/ssl-framework.adoc[SSL framework].
 
 Configuration for the backup server should be added to the _neo4j.conf_ file and configuration for backup client to the _neo4j-admin.conf_ file.
-SSL settings should be set identically between both to ensure compatibility.
+The easiest way to ensure compatibility is to use the same SSL policy configuration for both the server and the client.
+However, this approach is not recommended for production environments, where it is better to use Certificate Authorities (CAs) to sign the certificates used by both the server and the client.
```

Comment:
I'm unclear what this warning is about.
Using CAs to sign certificates is indeed a better, more secure method of using TLS but that can be done even when both the server and client use the same SSL policy configuration.